### PR TITLE
[wdpsec] add a test for the output only userAgent capability

### PR DESCRIPTION
**Pull request by OrKoN**
Spec change: https://github.com/w3c/webdriver/pull/1790

### CODE REVIEW

**Review by sadym-chromium — APPROVED** [review]

File: webdriver/tests/classic/new_session/response.py [hunk 1]
```diff
@@ -42,3 +42,10 @@ def test_capability_default_value(session, capability, default_value):
     assert isinstance(session.capabilities, dict)
     assert capability in session.capabilities
     assert session.capabilities[capability] == default_value
+
+
+def test_capability_user_agent(session):
+    assert isinstance(session.capabilities, dict)
+    assert "userAgent" in session.capabilities
+    assert type(session.capabilities["userAgent"]) == str
+    assert len(session.capabilities["userAgent"]) > 0
```
Comment (whimboo) [inline webdriver/tests/classic/new_session/response.py:51]: All the browsers that have WebDriver implemented basically have a user agent value that is starting with `Mozilla/`. Probably we can have a check instead to verify that?

Comment (whimboo) [reply webdriver/tests/classic/new_session/response.py:51]: Fair point. Lets leave it as is then.

**Review by whimboo — APPROVED** [review]
Looks fine to me. The stability job fails because we do not support this capability yet.

**Comment by OrKoN** [comment]
@web-platform-tests/admins Could someone please force merge the PR? (the stability issue is not related to the new test, some runs in Firefox has timed out).

**Pull request merged**